test/e2e/framework/util.go

```diff
@@ -1932,6 +1932,17 @@ func ExpectNoError(err error, explain ...interface{}) {
 	ExpectWithOffset(1, err).NotTo(HaveOccurred(), explain...)
 }
 
+func ExpectNoErrorWithRetries(fn func() error, maxRetries int, explain ...interface{}) {
+	var err error
+	for i := 0; i < maxRetries; i++ {
+		err = fn()
+		if err != nil {
+			Logf("(Attempt %d of %d) Unexpected error occurred: %v", i+1, maxRetries, err)
+		}
+	}
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), explain...)
```

Comment:
I mean the Expect... line logs if there's an error. I'm not sure if it logs it inline with a timestamp though...